config/locales/en.json

```diff
@@ -313,24 +313,40 @@
-	"section2.consent_valid_period": "This signed consent is valid for up to 3 years unless you cancel it in writing. Service Canada requires your original signature, but we will use a photocopy and consider it as valid as the original when requesting personal information from the persons and organizations listed above.",
+	"consent.preamble": "Service Canada is authorized under Section 68 and 69 of the Canada Pension Plan (CPP) Regulations to receive personal information (medical and non-medical) about you to determine if you qualify or continue to qualify for CPP disability benefits. Your consent to permit Service Canada to obtain this information is necessary, should Service Canada need this information from persons and organizations listed below.",
+	"consent.may_contact_preamble": "For this reason, Service Canada may contact any of the following persons and organizations if necessary:",
+	"consent.practioners": "medical doctors, nurse practitioners, consultant specialists, or other health care professionals",
+	"consent.facilities": "medical facilities or hospitals",
+	"consent.government": "federal, provincial, territorial, or municipal government departments and agencies",
+	"consent.compensation_boards": "provincial or territorial workers' compensation boards",
+	"consent.employers": "employers, former employers",
+	"consent.insurers": "administators of insurance plans",
+	"consent.education": "Educational institutions or other vocational agencies",
+	"consent.accountant": "My accountant or bookkeeper for information on self-employment",
+	"consent.financial": "Financial institutions (for address updates only)",
+	"consent.volunteer": "Volunteer organizations",
+	"consent.employees": "Employees (for cases of self-employed persons)",
+	"consent.optional_parties_preamble": "Service Canada may also need to contact the organizations and persons listed below. Deselect any organizations and persons that Service Canada may not contact",
+	"consent.question": "Do you give your consent to Service Canada to obtain medical and other personal informtion about you from the persons and organizations listed above?",
+	"consent.valid_period": "Your consent is valid for up to 3 years unless you cancel it ",
+	"consent.cancel_in_writing": "in writing to Service Canada.",
+	"consent.cancel_consent_link": "https://catalogue.servicecanada.gc.ca/content/EForms/en/ReturningtheForm/IspCpp.html",
+	"consent.i_understand_intro": "I understand that if I do not give my consent, Service Canada:",
+	"consent.i_understand_1": "will make a decision based on the available information on my file",
+	"consent.i_understand_2": "may stop paying me the benefits if I am already receiving them",
```

Comment:
This line should read: `may stop paying me the benefits if I am already receiving them; and`
Again, not my choice.